manim/utils/cfgwriter.py

```diff
@@ -15,8 +15,10 @@
 from rich.style import Style
 from rich.errors import StyleSyntaxError
 
+INVALID_STYLE_MSG = "[red bold]Your Style is not valid. Try again.[/red bold]"
 
```

Comment:
Issue: this file has a bunch of functions that are not meant to be used by outside users of the file. For example, `replace_keys` and `is_valid_style`. For this purpose, please add the `__all__` variable at the top of the file specifying which variables/functions this file exports.

(In the case when `is_valid_style` becomes useful for an outside user, then it should be moved to a utils file instead. But let's keep it here for now.)